core/src/saros/account/XMPPAccountStore.java

```diff
@@ -543,7 +585,9 @@ public synchronized boolean exists(String username, String domain, String server
    * @param jidString the jid of the user as string
    * @return the matching XMPP account or null in case of no match
    * @throws NullPointerException if jidString is null
+   * @deprecated Will be removed soon, do not use
```

Comment:
Refer to alternative methods?

Reply:
Just a riddle for you. However it is not complicated, I offered two get methods. One should solve your needs.